client/js/components/tags/TagListEditForm.vue

```diff
@@ -0,0 +1,154 @@
+
+<template>
+  <div class="flex">
+    <dp-input
+      v-if="isInEditState === nodeElement.id"
+      :id="`edit-${type}-${nodeElement.id}`"
+      class="flex-1"
+      v-model="unsavedItem.title" />
+    <template v-else>
+      <div
+        class="flex-1"
+        v-text="nodeElement.attributes.title" />
+
+      <div class="text-center w-9">
+        <dp-contextual-help
+          v-if="nodeElement.relationships?.boilerplate"
+          icon="file"
+          :text="nodeElement.relationships.boilerplate.attributes.title" />
+      </div>
+
+      <addon-wrapper
+        hook-name="tag.edit.form"
+        :addon-props="{
+          tag: nodeElement
+        }" />
```

Comment:
some styling nitpick: i think we have to maintain a consistent code styling, so if you add an empty lines in this block, it has to be added below as well

```suggestion
    <template v-else>
      <div
        class="flex-1"
        v-text="nodeElement.attributes.title" />
      <div class="text-center w-9">
        <dp-contextual-help
          v-if="nodeElement.relationships?.boilerplate"
          icon="file"
          :text="nodeElement.relationships.boilerplate.attributes.title" />
      </div>
      <addon-wrapper
        hook-name="tag.edit.form"
        :addon-props="{
          tag: nodeElement
        }" />
```